src/ec/packages/ec-component-hero-banner/hero-banner.html.twig

```diff
@@ -0,0 +1,78 @@
+{% spaceless %}
+{#
+  Parameters:
+    - "type" (string) (default: 'default') Type of banner (can be 'default','image','image-shade','primary')
+    - "title" (string) (default: '') Title of banner
+    - "image" (string) (default: '') Image for banner (required for image banner type)
+    - "description" (string) (default: '') Description of banner
+    - "centered" (bool) (default: true) Define if banner should be centered
+    - "button" (associative array) (default: predefined structure) predefined structure for EC Button component
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{# variables #}
+{% set _type = type|default('default') %}
+{% set _centered = centered|default(true) %}
+{% set _title = title|default('') %}
+{% set _description = description|default('') %}
+{% set _image = image|default('') %}
+
+{% set _button = {
+  variant: 'call',
+  icon: {
```

Comment:
the icon should come from the data, not from the template. Icon may vary depending on context (or may not be present at all)

so there is no need to set value for it there

Reply:
Check my answer https://github.com/ec-europa/ecl-twig/pull/49#discussion_r262992520